src/clj/puppetlabs/services/config/puppet_server_config_service.clj

```diff
@@ -26,7 +26,9 @@
           "Initializing with the following settings from core Puppet:\n%s"
           (ks/pprint-to-string puppet-config))
         (core/init-webserver! override-webserver-settings!
-                              (get-in tk-config [:webserver])
+                              (get-in tk-config
+                                      [:webserver :puppet-server]
+                                      (get-in tk-config [:webserver]))
                               puppet-config)
```

Comment:
@camlow325 Would that involve hard-coding the fully-qualified `master-service` namespace into this namespace?  That sounds a little gross.

But ... agreed that it would be nice to not hard-code the `:puppet-server` server name here.